src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -121,6 +128,11 @@ public int Size
                         Cosigners.GetVarSize() +    //Cosigners
                         Script.GetVarSize() +       //Script
                         Witnesses.GetVarSize();     //Witnesses
+
+                    if (Version == TransactionVersion.OracleResponse)
+                    {
```

Comment:
`TransactionType` will be better? It seems we are misusing version and type?  @roman-khimov I think this [comment](https://github.com/neo-project/neo/issues/1274#issuecomment-605117812) makes sense.

Reply:
True, types are types and versions are versions, both have their own valid meaning and purpose and it's better not to mix them.

Reply:
The original idea was this https://github.com/neo-project/neo/pull/1539#issuecomment-609378672